persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_36/generated/rainier/persist_client.bal

```diff
@@ -0,0 +1,282 @@
+        'class: "io.ballerina.stdlib.persist.QueryProcessor",
+        name: "query"
+    } external;
+
+    isolated resource function get 'departments/[string deptNo]('DepartmentTargetType targetType = <>) returns targetType|persist:Error = @java:Method {
+        'class: "io.ballerina.stdlib.persist.QueryProcessor",
+        name: "queryOne"
+    } external;
+
+    isolated resource function post 'departments('DepartmentInsert[] data) returns string[]|persist:Error {
+        _ = check self.persistClients.get('DEPARTMENT).runBatchInsertQuery(data);
+        return from 'DepartmentInsert inserted in data
+            select inserted.deptNo;
+    }
+
+    isolated resource function put 'departments/[string deptNo]('DepartmentUpdate value) returns 'Department|persist:Error {
+        _ = check self.persistClients.get('DEPARTMENT).runUpdateQuery(deptNo, value);
+        return self->/'departments/[deptNo].get();
+    }
+
+    isolated resource function delete 'departments/[string deptNo]() returns 'Department|persist:Error {
+        'Department result = check self->/'departments/[deptNo].get();
+        _ = check self.persistClients.get('DEPARTMENT).runDeleteQuery(deptNo);
+        return result;
+    }
+
+    isolated resource function get 'employees('EmployeeTargetType targetType = <>) returns stream<targetType, persist:Error?> = @java:Method {
+        'class: "io.ballerina.stdlib.persist.QueryProcessor",
+        name: "query"
+    } external;
+
+    isolated resource function get 'employees/[string empNo]('EmployeeTargetType targetType = <>) returns targetType|persist:Error = @java:Method {
+        'class: "io.ballerina.stdlib.persist.QueryProcessor",
+        name: "queryOne"
+    } external;
+
+    isolated resource function post 'employees('EmployeeInsert[] data) returns string[]|persist:Error {
+        _ = check self.persistClients.get('EMPLOYEE).runBatchInsertQuery(data);
+        return from 'EmployeeInsert inserted in data
+            select inserted.empNo;
+    }
+
+    isolated resource function put 'employees/[string empNo]('EmployeeUpdate value) returns 'Employee|persist:Error {
+        _ = check self.persistClients.get('EMPLOYEE).runUpdateQuery(empNo, value);
+        return self->/'employees/[empNo].get();
+    }
+
+    isolated resource function delete 'employees/[string empNo]() returns 'Employee|persist:Error {
+        'Employee result = check self->/'employees/[empNo].get();
+        _ = check self.persistClients.get('EMPLOYEE).runDeleteQuery(empNo);
+        return result;
+    }
+
+    isolated resource function get 'orderitems('OrderItemTargetType targetType = <>) returns stream<targetType, persist:Error?> = @java:Method {
+        'class: "io.ballerina.stdlib.persist.QueryProcessor",
+        name: "query"
+    } external;
+
+    isolated resource function get 'orderitems/[string itemId, string orderId] ('OrderItemTargetType targetType = <>) returns targetType|persist:Error = @java:Method {
```

Comment:
Is this defined path correct?

Reply:
updated in [39a2eee](https://github.com/ballerina-platform/persist-tools/pull/169/commits/39a2eee898b0a4e298b485775ff88cca0df16c69) and [39a2eee](https://github.com/ballerina-platform/persist-tools/pull/169/commits/39a2eee898b0a4e298b485775ff88cca0df16c69)